lua/starfall/libs_cl/render.lua

```diff
@@ -1867,6 +1867,24 @@ function render_library.drawSimpleText(x, y, text, xalign, yalign)
 	return draw.SimpleText(text, font, x, y, currentcolor, xalign, yalign)
 end
 
+--- Draws outlined text more easily but no new lines or tabs.
```

Comment:
I looked in the source for this and I think it does do new lines and tabs.

Reply:
```lua
hook.Add("HUDPaint", "test", function()
    draw.SimpleTextOutlined( 
        "Te\nst Hel\tlo",
        "DermaDefault",
        ScrW() / 2,
        ScrH() / 2,
        Color( 255, 255, 255, 255 ),
        TEXT_ALIGN_LEFT,
        TEXT_ALIGN_TOP,
        2,
        Color( 255, 0, 0, 255 )
    )
end)
```
![image](https://github.com/user-attachments/assets/c38121c2-88fb-41da-9602-5cff70fb1b91)

SimpleTextOutlined internally use `SimpleText( text, font, x, y, colour, xalign, yalign )`
and state that it dosn't handle newline correctly
https://wiki.facepunch.com/gmod/draw.SimpleText

Reply:
Ah ok, I was looking at DrawText instead of SimpleText